server/plugin.ts

```diff
@@ -126,6 +134,14 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
       setupMultitenantRoutes(router, securitySessionStorageFactory, this.securityClient);
     }
 
+    if (config.multitenancy.enable_aggregation_view) {
+      core.savedObjects.addClientWrapper(
+        1,
```

Comment:
OSD data source support feature added a saved object client wrapper with order `1` at https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source/server/plugin.ts#L62-L66 . which will cause conflict. Let's choose another order for this wrapper

Reply:
changed the priority to `2`